sdk/tools/azure-sdk-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>jar</packaging>
+
+    <name>${artifactId}</name>
+
+    <properties>
+        <maven.compiler.target>${javaVersion}</maven.compiler.target>
+        <maven.compiler.source>${javaVersion}</maven.compiler.source>
+        <bom.version>1.0.2</bom.version>
```

Comment:
The latest version of the BOM is `1.0.5`, is there any way to have the archetype look this up from Maven central rather than needing to release each time a new version of the BOM is released?

Reply:
Yes, this is just the base version that is hard-coded in the template. The `archetype-post-generate.groovy` script runs later to update the version to the latest available version in Maven central.